README.md

```diff
@@ -167,7 +167,7 @@ first sequence has fewer, the same or more elements than the second sequence.
 
 Returns a sequence consisting of the head element and the given tail elements.
 
-This method is obsolete and will be removed in a future version. Use `Append`
+This method was removed with version 4.0. Use `Append`
 instead.
```

Comment:
Would be worth calling out to use `Append` from .NET (the one from MoreLINQ may be removed as part of PR #945):

```suggestion
This extension was rendered obsolete in version 3.0 and eventually removed in
version 4.0. Use [`Append`][linq-append] from .NET instead that's been available
since .NET Standard 1.6+, .NET Core 1.0+ and .NET Framework 4.7.1+.

[linq-append]: https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.append
```